packages/shoreline/src/components/search/search.tsx

```diff
@@ -26,6 +30,8 @@ export const Search = forwardRef<HTMLInputElement, SearchProps>(
       ...inputProps
     } = props
 
+    const getMessage = useMessage(placeholder ? { placeholder } : undefined)
```

Comment:
I've noticed that this step is missing in the documentation and I just created an Issue to map that: https://github.com/vtex/shoreline/issues/1777

Reply:
I understand that the standard is to have a messages prop, but you used this in complex components (confirmationModal, collectionView, ...).
In this case of search, the placeholder is a native prop of the input. So to follow the standard, should I omit a standard HTML prop? Or should I keep it and then it can be overridden in two ways?
I don't find either of the options interesting.

Reply:
IMO its ok to omit the prop to keep the design system consistency. Also, search is not a native component.